Part1.md

```diff
@@ -18,10 +18,31 @@ Rails 5 or 6 but is not guaranteed to.)
 You will also need the `bundler` gem.  If `bundle -v` fails, `gem install bundler`
 to install it.
 
+## IMPORTANT: Issues with Bundler and Rails versions in Codio
+
+The current (2020-10-01) version of the Codio stack contains a version
+of Bundler that doesn't work with this assignment, and the wrong
+version of Rails for using `rails new` to generate an app skeleton.  We're working on
+updating Codio but in the meantime, do these steps manually:
+
+```
+sudo gem uninstall --version '>=0' --all bundler
+sudo gem uninstall --version '> 4' --all rails
+sudo gem uninstall --version '> 4' --all railties
+sudo gem install --no-document --version '1.17.3' bundler
+sudo gem install --no-document --version '4.2.10' rails
+```
+
+(`--no-ri` skips the lengthy step of installing local documentation
+for the Gems; it's much faster to use the online documentation.  Also,
+you don't have to explicitly reinstall `railties` because it will be
+installed as a dependency of `rails`.)
+
 ## Create a new Rails app
 
 Now that you have Ruby and Rails installed, create a new, empty
-Rails app with the command `rails new rottenpotatoes --skip-test-unit --skip-turbolinks`.
+Rails app with the command `rails new rottenpotatoes --skip-test-unit
```

Comment:
````suggestion
Rails app with the command: 
```sh
rails new rottenpotatoes --skip-test-unit
```
````

IMO, given the importance of the command it's helpful to see the as a larger code block.